fv3/fv3_cap.F90

```diff
@@ -394,6 +395,10 @@ subroutine InitializeAdvertise(gcomp, rc)
       enddo
 
       call ESMF_ConfigGetAttribute(config=CF, value=time_unlimited, label ='time_unlimited:', default=.false., rc=rc)
+      if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
+
+      call ESMF_ConfigGetAttribute(config=CF, value=sync_fcst_info_to_wgc, label ='sync_fcst_info_to_wgc:', default=.false., rc=rc)
```

Comment:
Add comments: !! sync_fcst_info_to_wgc: flag to synchronize EMSF_info from fcst grid comp to write grid comps. Needs ESMF update to allow this to be done with write grid component independently.

Reply:
Added comment in https://github.com/NOAA-EMC/fv3atm/pull/958/commits/27c5b585fdea06bdb9b2dd77d19f7ddc2e3281c2